contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3285,15 +3285,44 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 								{
 									RoleSpec   *rolspec = lfirst(item);
 									char	   *user_name;
+									char       *db_principal;
+									const char *db_owner_name;
+									int	       role_oid;
+									int        rolename_len;
 
 									user_name = get_physical_user_name(db_name, rolspec->rolename, false);
+									db_owner_name = get_db_owner_name(get_cur_db_name());
+									role_oid = get_role_oid(user_name, true);
```

Comment:
why missing_ok = true?

Reply:
For if exist case which is already present in some testfiles